tests/ui/split_with_space.rs

```diff
@@ -0,0 +1,16 @@
+//@aux-build:proc_macros.rs
```

Comment:
This test links proc_macros.rs but never actually uses it.

It probably should, though, so this is a good idea. Can you add a test with proc macros? You can look at e.g. [this file](https://github.com/rust-lang/rust-clippy/blob/f2c74e220bef89dc62dfbd1d2a4eb49c6ae8b47a/tests/ui/arc_with_non_send_sync.rs#L21) to see how to write a test that uses this.
You'll also likely need to add a `!is_from_proc_macro()` check to the lint code.